.circleci/config.yml

```diff
@@ -55,6 +55,17 @@ jobs:
           root: .
           paths:
             - node_modules
+  eslint:
+    docker:
+      - image: "cimg/node:lts"
+    resource_class: small
+    steps:
+      - checkout
+      - attach_workspace:
+          at: .
+      - run:
+          name: Lint static CSS, JS & YAML files
+          command: npx eslint . --no-error-on-unmatched-pattern
```

Comment:
The flag should probably be removed once the first actual JS/TS file is added, but without the CircleCI checks for this branch will not pass 😅